lib/rules/no-class-bindings.js

```diff
@@ -1,27 +1,183 @@
 'use strict';
 
+const { builders: b } = require('ember-template-recast');
+
 const Rule = require('./base');
 
+// from
+// https://github.com/emberjs/ember.js/blob/v3.25.1/packages/@ember/string/index.ts#L12-L16
+// https://github.com/emberjs/ember.js/blob/v3.25.1/packages/@ember/string/index.ts#L63-L67
+
+const STRING_DECAMELIZE_REGEXP = /([\da-z])([A-Z])/g;
+const STRING_DASHERIZE_REGEXP = /[ _]/g;
+
+function dasherize(str) {
+  let decamelizedString = str.replace(STRING_DECAMELIZE_REGEXP, '$1_$2').toLowerCase();
+
+  return decamelizedString.replace(STRING_DASHERIZE_REGEXP, '-');
+}
+
 module.exports = class NoClassBindings extends Rule {
   visitor() {
-    function check(node) {
+    let check = (node) => {
```

Comment:
Why was this necessary?

Reply:
The function uses `this` internally. Previously, it worked because the visitor hooks themselves are called bound to the plugin instance but this refactor changed to _sometimes_ calling this function directly (e.g. when it wouldn't be bound to the plugin instance naturally). Changing to an arrow function meant that I didn't have to update to `check.call(this, node)` in the fixers below.